README.md

```diff
@@ -1,38 +1,53 @@
 
-### Core services
+These preview libraries can be easily identified by their folder, package, and namespaces names starting with 'Azure', e.g. Azure.Storage.Blobs. 
 
+The libraries released in the July 2019 preview:
+* [Azure.ApplicationModel.Configuration](/sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md)
 * [Azure.Messaging.EventHubs](/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
 * [Azure.Identity.KeyVault.Keys](/sdk/keyvault/Azure.Security.KeyVault.Keys/Readme.md)
 * [Azure.Identity.KeyVault.Secrets](/sdk/keyvault/Azure.Security.KeyVault.Secrets/Readme.md)
 * [Azure.Storage.Blobs](/sdk/storage/Azure.Storage.Blobs/README.md)
 * [Azure.Storage.Files](/sdk/storage/Azure.Storage.Files/README.md)
 * [Azure.Storage.Queues](/sdk/storage/Azure.Storage.Queues/README.md)
+* [Azure.Core](/sdk/core/Azure.Core/README.md) (shared library)
 
-### Shared libraries
+>NOTE: If you need to ensure your code is ready for production, use one of the stable libraries.
 
-Azure SDK clients use shared libraries implementing retries, logging, transport protocols, authentication protocols, etc.
+### Client: Stable
+Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the preview packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. Stable library directories typicaly contain 'Microsoft.Azure' in their names, e.g. 'Microsoft.Azure.KeyVault'.
 
-[Azure.Core](/sdk/core/Azure.Core/README.md)
+### Management
+Libraries which enable you to provision specific server resources. They are directly mirroring Azure service's REST endpoints. Management library directories typicaly contain the word 'Management' in their names, e.g. 'Microsoft.Azure.Management.Storage'.
 
-### Other services
-
-[Azure.ApplicationModel.Configuration](/sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md)
+## Need help?
+* For reference documentation visit the [Azure SDK for .NET API Reference](http://aka.ms/net-docs).
+* For tutorials, samples, quick starts, and other documentation, go to [Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/azure/).
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-net/issues/new/choose).
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure-net-sdk) or ask new ones on StackOverflow using `azure-net-sdk` tag.
 
 ## Contributing
-
 For details on contributing to this repository, see the [contributing guide](CONTRIBUTING.md).
+
+| Component | Build Status |
```

Comment:
I moved it because it seems like it's information more relevant to contributors, and not really granular enough to be useful. But if we all think it should be at the top, I will move it.